### ISSUE TITLE
Breaking changes in 2.0.0?

### ISSUE THREAD

**Issue by gmaclennan** [sindresorhus/camelcase#9]
What are the breaking changes in 2.0.0?

**Comment by sindresorhus** [comment]
https://github.com/sindresorhus/camelcase/commit/31039cd82fe0bb6b8ceceae85d3524ecdb03f91e

You could have easily checked the commit log yourself...

**Comment by gmaclennan** [comment]
My apologies. I did look for `CHANGES.md`, I looked through the releases, I looked through all the issues, I looked through all the Pull Requests, and I did look at the commit log. I could not tell from just looking at the commit messages which one was a breaking change. I guess if I had looked at each commit and read the code I could have found out...

For anybody else who uses this library and does not have much time for reading through commit logs of every library you use: v2.0.0 will no longer change the case of single character strings.

```js
// v1.2.1
camelCase('F')
//=> 'f'

// v2.0.0
camelCase('F')
//=> 'F'
```
